pkg/webhook/certs.go

```diff
@@ -130,10 +128,10 @@ func createCA(ctx context.Context) (*rsa.PrivateKey, *x509.Certificate, []byte,
 // key for the server. serverKey and serverCert are used by the server
 // to establish trust for clients, CA certificate is used by the
 // client to verify the server authentication chain.
-func CreateCerts(ctx context.Context) (serverKey, serverCert, caCert []byte, err error) {
+func CreateCerts(ctx context.Context, name, namespace string) (serverKey, serverCert, caCert []byte, err error) {
```

Comment:
would be nice to pull these several certs functions into an object `Certs{Ctx: ctx, Name: name, Namespace: ns}`.

Reply:
Yeah, I've been thinking about ways to refactor this code more generally for a while.

This refactoring is just to support moving it to `knative/pkg`.  I should make this private in that move, so that it's easier to do internal breaking refactorings without worrying about folks depending on the guts (this signature as-is)